segment.go

```diff
@@ -22,29 +22,25 @@ const (
 	segmentValidateEndpoint       = "segment/validate"
 )
 
+// Segment is a logical expression to filter entity
+//  The normal concept is logical filter to find users, but the
+//  table also allows logical filter on content, or other entity types.
 type Segment struct {
 	Id            string    `json:"id"`
-	Aid           int       `json:"aid"` // Deprecated; use AccountId
 	AccountId     string    `json:"account_id"`
-	ShortId       string    `json:"short_id,omitempty"`
 	Name          string    `json:"name"`
 	IsPublic      bool      `json:"is_public"`
-	PublicName    string    `json:"public_name,omitempty"`
 	SlugName      string    `json:"slug_name"`
 	Description   string    `json:"description,omitempty"`
 	SegKind       string    `json:"kind,omitempty"`
 	Table         string    `json:"table,omitempty"`
 	AuthorId      string    `json:"author_id"`
-	Updated       time.Time `json:"updated" bson:"updated"`
-	Created       time.Time `json:"created" bson:"created"`
-	SegType       string    `json:"op"`
-	Negate        bool      `json:"negate"`
+	Updated       time.Time `json:"updated"`
+	Created       time.Time `json:"created"`
 	Tags          []string  `json:"tags"`
 	Category      string    `json:category,omitempty`
 	Invalid       bool      `json:"invalid"`
 	InvalidReason string    `json:"invalid_reason"`
-	Deleted       bool      `json:"deleted"`
```

Comment:
that isn't intended as external attribute is it?   think it isn't even returned in api